config/components.php

```diff
@@ -140,9 +140,16 @@
 	'search' => function (
 		App $kirby,
 		Collection $collection,
-		string $query = '',
-		$params = []
+		string $query = null,
```

Comment:
This should still primarily be a string IMO, but a nullable one:

```suggestion
		string|null $query = '',
```